src/analysis/retail/warrior/arms/modules/checklist/Component.tsx

```diff
@@ -61,42 +61,43 @@ const ArmsWarriorChecklist = ({
+            //removed: <SpellLink id={SPELLS.MASSACRE_TALENT_ARMS.id} />) . It should only be used during the
+            //execution phase to refresh{' '}
             <>
-              You should cast <SpellLink id={SPELLS.MORTAL_STRIKE.id} icon /> as much as possible
-              when the target is above 20% (or 35% with{' '}
-              <SpellLink id={SPELLS.MASSACRE_TALENT_ARMS.id} />) . It should only be used during the
-              execution phase to refresh{' '}
-              <SpellLink id={SPELLS.MASTERY_DEEP_WOUNDS_DEBUFF.id} icon /> as{' '}
-              <SpellLink id={SPELLS.EXECUTE.id} /> is more rage efficient than Mortal Strike.
+              Try to use as many <SpellLink id={SPELLS.MORTAL_STRIKE.id} icon /> as possible. It is
+              generally your strongest hitting ability, unless you are in{' '}
+              <SpellLink id={SPELLS.EXECUTE} /> range and do not have the{' '}
+              <SpellLink id={TALENTS.EXECUTIONERS_PRECISION_TALENT} /> talent. Especially with the
+              4-set bonus in Season 1 of Dragonflight, where keeping up your buff up is important.
             </>
           }
         >
           <Requirement
             name={
               <>
-                <SpellLink id={SPELLS.MORTAL_STRIKE.id} icon /> outside execution phase
+                <SpellLink id={SPELLS.MORTAL_STRIKE.id} icon /> uses
               </>
             }
-            thresholds={thresholds.goodMortalStrike}
-          />
-          <Requirement
-            name={
-              <>
-                <SpellLink id={SPELLS.MORTAL_STRIKE.id} icon /> which should have been{' '}
-                <SpellLink id={SPELLS.EXECUTE.id} icon />
-              </>
-            }
-            thresholds={thresholds.tooMuchMortalStrike}
-          />
-          <Requirement
-            name={
-              <>
-                <SpellLink id={SPELLS.MASTERY_DEEP_WOUNDS_DEBUFF.id} icon /> refreshed with{' '}
-                <SpellLink id={SPELLS.MORTAL_STRIKE.id} icon /> during execution phase
-              </>
-            }
-            thresholds={thresholds.notEnoughMortalStrike}
+            thresholds={thresholds.mortalStrikeUsage}
           />
+          {/*<Requirement*/}
+          {/*  name={*/}
+          {/*    <>*/}
+          {/*      <SpellLink id={SPELLS.MORTAL_STRIKE.id} icon /> which should have been{' '}*/}
+          {/*      <SpellLink id={SPELLS.EXECUTE.id} icon />*/}
+          {/*    </>*/}
+          {/*  }*/}
+          {/*  thresholds={thresholds.tooMuchMortalStrike}*/}
+          {/*/>*/}
+          {/*<Requirement*/}
+          {/*  name={*/}
+          {/*    <>*/}
+          {/*      <SpellLink id={SPELLS.MASTERY_DEEP_WOUNDS_DEBUFF.id} icon /> refreshed with{' '}*/}
+          {/*      <SpellLink id={SPELLS.MORTAL_STRIKE.id} icon /> during execution phase*/}
+          {/*    </>*/}
+          {/*  }*/}
+          {/*  thresholds={thresholds.notEnoughMortalStrike}*/}
+          {/*/>*/}
```

Comment:
if these suggestions are no longer correct for modern arms, probably just delete.

```suggestion

```